package.json

```diff
@@ -50,7 +50,7 @@
   "dependencies": {
     "@material-ui/lab": "4.0.0-alpha.61",
     "classnames": "2.3.1",
-    "cozy-client": "^32.8.0",
+    "cozy-client": "^33.0.8",
```

Comment:
what is the purpose of this commit?
could you get inspired by this commit and this PR, to document the purpose of the upgrade, please? https://github.com/cozy/cozy-drive/commit/e375cd8b480eef906e92e6e5726c889d649ad093